en/tutorials-and-examples/cms/articles-controller.rst

```diff
@@ -184,13 +184,12 @@ to be created. Start by creating an ``add()`` action in the
         {
             parent::initialize();
 
-            $this->loadComponent('Paginator');
-            $this->loadComponent('Flash'); // Include the FlashComponent
+            $this->set('articles', $this->paginate($this->Articles));
```

Comment:
I don't think this is correct. We should remove the `initialize()` method in this example in total because the component loading already happens in the AppController.

And paginating a model should happen inside each sub-controller associated to their model

Reply:
Removed